sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -172,4 +173,22 @@ export class AssessmentAESTController extends BaseController {
   ): Promise<AssessmentNOAAPIOutDTO> {
     return this.assessmentControllerService.getAssessmentNOA(assessmentId);
   }
+
+  /**
+   * Get estimated and actual(if present) award details of an assessment.
+   * @param assessmentId assessment to which awards details belong to.
+   * @returns estimated and actual award details.
+   */
+  @Get(":assessmentId/award")
+  @ApiNotFoundResponse({
+    description: "Assessment not found.",
+  })
+  async getAssessmentAwardDetails(
+    @Param("assessmentId", ParseIntPipe) assessmentId: number,
+  ): Promise<AwardDetailsAPIOutDTO> {
+    return this.assessmentControllerService.getAssessmentAwardDetails(
```

Comment:
await ?

Reply:
Await is not required here as inside this.assessmentControllerService.getAssessmentAwardDetails there is await and it just returns values from there.